GEMINI.md

```
@@ -153,41 +153,83 @@ println!("Connecting to database...");
-default = ["json", "csv", "ssl", "additional_mysql_types", "verbose"]
-ssl = ["mysql/native-tls"]                 # Platform native TLS (default)
-ssl-rustls = ["mysql/rustls-tls"]         # Pure Rust TLS (alternative)
+default = ["json", "csv", "additional_mysql_types", "verbose"]
 additional_mysql_types = [...]             # BigDecimal, Decimal, etc.
 verbose = []                               # Conditional logging
 ```
 
+Note: TLS is now always available and is no longer a feature flag.
+
 **TLS Implementation Notes:**
 
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)